src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -538,7 +538,7 @@ private static bool CheckConstraintsSingleType(TypeSymbol type, CheckConstraints
             foreach (var underlyingTuple in underlyingTupleTypeChain)
             {
                 ArrayBuilder<TypeParameterDiagnosticInfo> useSiteDiagnosticsBuilder = null;
-                CheckTypeConstraints(underlyingTuple, conversions, currentCompilation, diagnosticsBuilder, ref useSiteDiagnosticsBuilder);
+                CheckTypeConstraints(underlyingTuple, conversions, currentCompilation, diagnosticsBuilder, diagnosticsBuilder, ref useSiteDiagnosticsBuilder);
```

Comment:
Consider naming the second `diagnosticsBuilder` argument (here and other locations).